go.mod

```diff
@@ -32,9 +33,10 @@ require (
 	github.com/golangci/golangci-lint v1.55.2
 	github.com/gorilla/mux v1.8.1
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.19.1
 	github.com/mgechev/revive v1.3.6
 	github.com/mitchellh/mapstructure v1.5.0
-	github.com/ojo-network/price-feeder v0.1.10-0.20240221234245-b3d7d675adc9
+	github.com/ojo-network/price-feeder v0.1.11-0.20240403153904-aeda5a5840a6
```

Comment:
Before we merge this in we should tag it to a specific price feeder release (after [this gets merged](https://github.com/ojo-network/price-feeder/pull/370))